### PR TITLE
Fixes issues #18 Address content refinement in Scatter Plots > Introduction

### PR DESCRIPTION
**Pull request by tmadruga**
The introduction in scatter plots does not really serve as an introduction to the section as a whole. I have created a new introduction, while renaming the old introduction old introduction.

### CODE REVIEW

**Review by raskutti — CHANGES_REQUESTED** [review]
See comment in #21.

**Comment by tmadruga** [comment]
@raskutti Changes from PR#21 addressed here.

File: ac1/_sources/scatter_plots_and_correlation/introduction.rst [hunk 1]
```diff
@@ -6,91 +6,19 @@
...
-
-.. shortanswer:: scatter_intro_q6
-
-   Dayton, Ohio and Memphis, Tennessee are both in the Central
-   region, but Dayton has a latitude of 39.54 degrees, while Memphis has a
-   latitude of 35.07 degrees. These cities were put in one category, even
-   though they have very different latitudes. Using what you’ve learned,
-   predict which of these cities (and why) will have a colder January
-   temperature.
+As of right now, you have learned about histograms and bar charts, which are
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/introduction.rst]: "As of right now" feels a bit awkward. Maybe just remove it and say "You have already learned about..."

File: ac1/_sources/scatter_plots_and_correlation/introduction.rst [hunk 2]
```diff
@@ -6,91 +6,19 @@
...
+data visualizations that are useful for understanding data. However, sometimes
+these visualizations are not the most intuitive for looking at individual pieces
+of a larger data set. This is where Scatter plots come in. Scatter plots are a
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/introduction.rst]: nit: Don't capitalize "scatter" when it's not the start of the sentence

File: ac1/_sources/scatter_plots_and_correlation/introduction.rst [hunk 3]
```diff
@@ -6,91 +6,19 @@
...
+type of data visualization that show the relationship between two quantitative
+variables. An example of a scatter plot is shown below. As you can see in the
+screenshot, each data point is plotted individually onto a graph. By organizing
+data this way, it becomes easier to understand trends in larger data sets.
+
+.. image:: figures/example_scatterplot.png
+   :align: center
+   :alt: An example of a spreadsheet
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/introduction.rst]: Alt text should be `An example of a scatter plot`

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+Motivating Scatterplots
+=======================
+
+Weather is something that changes based on location. Cities closer to the north
+and south pole are generally pretty cold, while cities close to the equator are
+generally pretty warm. But what if you wanted to quantify this relationship in
+order to make specific predictions? Instead of saying “cities are generally
+colder closer to the north pole,'' you could say, “as latitude increases by 10
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: nit: For some reason the end quotes came out as `''` but should be `"`.

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+degrees, temperature decreases by 20 degrees.”
+
+Given the following dataset with latitude, longitude, and weather data for
+different cities in the United States, you can use histograms to plot latitudes
+and temperatures independently as shown below.
+
+
+.. image:: figures/lat_temp_histograms.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst:21]: Missing `:align: center`

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+Since you know that temperature is related to location, you could group the
+cities based on their latitude, and then compare the mean temperature for each
+group. Comparing the means for each group will quantify if and how much latitude
+impacts temperature.
+
+A latitude closer to 0 degrees means a city is closer to the equator, and a
+latitude closer to 45 degrees means a city is closer to the poles. For cities in
+the northern hemisphere, like those in the USA, smaller latitudes indicate
+southern cities, and larger latitudes indicate northern cities. See here for a
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: See where for a map?

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+map of the USA with latitude and longitude lines.
+
+You can split up cities in the United States into three regions, “Northern,”
+“Southern,” and Central to more easily analyze the data. Most cities in the
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: Central is missing quotation marks

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+continental states are between 25 and 50 degrees, so to group these, you can
+label cities south of 35 degrees latitude “Southern”, cities north of 40 degrees
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst:43]: Throughout this file, you need to be consistent as to whether you put the commas and periods within the quotes or not. It doesn't matter too much which you choose (there are some examples throughout this whole file), but be consistent.

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+latitude “Northern”, and those in the middle “Central.” This choice is somewhat
+arbitrary, but it breaks the cities into three fairly even groups.
+
+Break down these cities into these three separate groups by using the IF
+function. Add a new column labelled location. Then create a formula as shown in
+the screenshot below to group the cities.
+
+
+.. image:: figures/city_region_breakdown.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst:52]: Missing align

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+map of the USA with latitude and longitude lines.
+
+You can split up cities in the United States into three regions, “Northern,”
+“Southern,” and Central to more easily analyze the data. Most cities in the
+continental states are between 25 and 50 degrees, so to group these, you can
+label cities south of 35 degrees latitude “Southern”, cities north of 40 degrees
+latitude “Northern”, and those in the middle “Central.” This choice is somewhat
+arbitrary, but it breaks the cities into three fairly even groups.
+
+Break down these cities into these three separate groups by using the IF
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: Here and throughout this file, use monospace text for Excel functions such as `IF` and `AVERAGEIF`.

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+.. image:: figures/city_region_breakdown.png
+   :alt: A screenshot of a Sheet with a formula to sort latitudes into the three regions.
+
+
+Which region do you think will have the highest January temperatures? What about
+the lowest July temperatures? Now, double check your predictions by finding the
+mean temperatures of each of the three regions. You can do this by using the
+AVERAGEIF function :ref:`(from the section on grouping)<grouping_data>`.
+
+.. image:: figures/avg_if.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst:62]: Missing align

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+
+
+.. image:: figures/city_region_breakdown.png
+   :alt: A screenshot of a Sheet with a formula to sort latitudes into the three regions.
+
+
+Which region do you think will have the highest January temperatures? What about
+the lowest July temperatures? Now, double check your predictions by finding the
+mean temperatures of each of the three regions. You can do this by using the
+AVERAGEIF function :ref:`(from the section on grouping)<grouping_data>`.
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: Just making sure that this internal reference works in Runestone?

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+
+     - Incorrect
+
+Functions like AVERAGEIF are a great way to calculate the mean for a specific
+region and/or month. But what if you want to see, in one visualization, the mean
+temperature for January and July across all regions? To do this, you can use a
+bar chart.
+
+
+.. image:: figures/avg_temp_region_jan.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst:136]: Missing align

File: ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+has a latitude of 39.54 degrees, while Memphis has a latitude of 35.07 degrees,
+which are quite different. Based on what you’ve learned, you could probably make
+an educated guess about which of these locations are colder in January, but how
+could you get a better prediction of what the temperature may be? This is where
+scatterplots will come in handy.
+
+
+.. _following dataset: https://drive.google.com/open?id=1Qhl09FAMxhHr9WVIKxfVK9UklvDVpiYMdgvKuaWZRa0
+.. _latitude, longitude: https://en.wikipedia.org/wiki/Geographic_coordinate_system
+.. See here for a map of the USA with latitude and longitude lines.: https://pasarelapr.com/images/map-of-usa-with-lines-of-latitude-and-longitude/map-of-usa-with-lines-of-latitude-and-longitude-2.jpg
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: Is this reference missing an underscore?

Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/motivating_scatterplots.rst]: Do these references work in Runestone? Seems to me like they are missing something to make them work.

**Review by raskutti — CHANGES_REQUESTED** [review]
This is great work, and well written. This review may look like a lot, but it is a lot of very small comments. The overall flow is much better, thank you!

Make sure the book compiles well and all images and links work in Runestone.

**Comment by raskutti** [comment]
@tmadruga minor changes requested

**Comment by tmadruga** [comment]
@raskutti changes made.

**Review by raskutti — APPROVED** [review]

**Pull request merged**